client/dashboard/emails/dataviews.tsx

```diff
@@ -72,19 +83,32 @@ export const emailFields: Field< Email >[] = [
 			if ( item.status === 'active' ) {
 				return <Text intent="success">{ __( 'Active' ) }</Text>;
 			}
-			if ( item.status === 'pending' ) {
+
+			if ( item.status === 'unverified_forwards' ) {
 				return <Text intent="warning">{ __( 'Pending verification' ) }</Text>;
 			}
+
+			if ( item.status === 'google_pending_tos_acceptance' ) {
+				return <Text intent="warning">{ __( 'Action required' ) }</Text>;
+			}
+
 			if ( item.status === 'suspended' ) {
 				return <Text intent="error">{ __( 'Expired' ) }</Text>;
 			}
+
+			// We can't handle un used mailboxes from a mailbox row because it's tied to the account.
+			if ( item.status === 'unused_mailboxes' ) {
+				return <Text intent="success">{ __( 'Active' ) }</Text>;
+			}
```

Comment:
We're going to need to figure out what to do when we have unused mailboxes. I'll create an issue for this and ping some designers.